data-serving/README.md

```diff
@@ -1,18 +1,78 @@
 
 # All things data
 
-## Case schema updates
-### What needs to be changed & where
+[toc]
+
+## Database design
+
+G.h case data, as well as the source, user, and session data for the curator portal, is stored in a MongoDB database.
+
+We have multiple instances of MongoDB, ranging from local instances for development, to dev (for
+http://dev-curator.ghdsi.org/), to prod prod (for http://dev-curator.ghdsi.org/).
```

Comment:
Extra `prod`? And prod URL shouldn't have `dev-`.

Reply:
sorta seems like I broke